src/core/api/auth/auth.types.ts

```diff
@@ -49,6 +49,14 @@ export interface AuthRes {
   token_type: 'Bearer';
 }
 
+export interface GoogleAuthRes {
```

Comment:
you don't need to duplicate type with `extent` -> `export interface GoogleAuthRes extend AuthRes`

Reply:
Done!